components/eventing-controller/pkg/handlers/utils.go

```diff
@@ -189,3 +195,59 @@ func GetRandString(l int) string {
 	}
 	return string(b)
 }
+
+func RemoveStatus(sub eventingv1alpha1.Subscription) *eventingv1alpha1.Subscription {
+	desiredSub := sub.DeepCopy()
+	desiredSub.Status = eventingv1alpha1.SubscriptionStatus{}
+	return desiredSub
+}
+
+func UpdateSubscription(ctx context.Context, dClient dynamic.Interface, sub *eventingv1alpha1.Subscription) error {
```

Comment:
The function name does not match what it actually do.
It should be renamed to `UpdateSubscriptionStatus`.

Reply:
Renamed